### PR TITLE
CSCC Snippets

### PR DESCRIPTION
**Pull request by emkornfield**
Add remaining snippets for CSCC

### CODE REVIEW

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/FindingSnippets.java [hunk 1]
```diff
@@ -0,0 +1,328 @@
+package com.google.cloud.examples.securitycenter.snippets;
```
Comment (sduskis) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/FindingSnippets.java]: Please add the standard header.

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SecurityMarkSnippets.java [hunk 2]
```diff
@@ -0,0 +1,238 @@
+package com.google.cloud.examples.securitycenter.snippets;
```
Comment (sduskis) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SecurityMarkSnippets.java]: Standard header.

Comment (emkornfield) [reply google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SecurityMarkSnippets.java]: done.

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/FindingSnippets.java (hunk 1 above)
Comment (emkornfield) [reply google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/FindingSnippets.java]: done.

**Comment by sduskis** [comment]
@emkornfield, you need to run `mvn com.coveo:fmt-maven-plugin:format` again.

**Comment by emkornfield** [comment]
Sorry about that.  done.

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/AssetSnippets.java [hunk 3]
```diff
@@ -175,7 +180,130 @@ private AssetSnippets() {}
       throw new RuntimeException("Couldn't create client.", e);
     }
   }
-  // [END list_asset_changes_status_changes]
+  // [END list_asset_changes_status_changes]\
```
Comment (sduskis) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/AssetSnippets.java]: Please remove the trailing `\`

**Pull request merged**